backend/src/v5/processors/teamspaces/projects/models/commons/tickets.groups.js

```diff
@@ -79,22 +105,126 @@ const getObjectArrayFromRules = async (teamspace, project, model, revId, rules)
 	return { container: model, _ids: Object.values(matchedMeshes) };
 };
 
-TicketGroups.updateTicketGroup = updateGroup;
+const convert3dRepoIdsToExternalIds = async (teamspace, project, objects) => {
+	const convertedObjects = await Promise.all(objects.map(async (obj) => {
+		if (!obj._ids) {
+			return obj;
+		}
+
+		const convertedObject = { ...obj };
+
+		const shared_ids = await getNodesByIds(teamspace, project, obj.container, obj._ids,
```

Comment:
the context of this variable is an array of  node objects,  so `shared_ids` being the variable name is quite confusing. Should rename to `nodes` or something